src/kakarot/accounts/eoa/library.cairo

```diff
@@ -90,6 +90,7 @@ namespace ExternallyOwnedAccount {
         bitwise_ptr: BitwiseBuiltin*,
         range_check_ptr,
     }(call_array_len: felt, call_array: CallArray*, calldata_len: felt, calldata: felt*) -> () {
+        alloc_locals;
```

Comment:
why do we need this?

Reply:
it created revoked references for functions calling this one without defining exactly the same implicit args 👀